test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -86,7 +90,8 @@ public async Task TempFilter()
             EdgeDeployment deployment = await this.runtime.DeployConfigurationAsync(
                 builder =>
                 {
-                    builder.AddModule(tempSensorModule.Name, tempSensorModule.Image);
+                    builder.AddModule(tempSensorModule.Name, tempSensorModule.Image)
+                        .WithEnvironment(new[] { ("MessageCount", "1") });
```

Comment:
Woops, I forgot about `TempFilterFunc()`. It needs to be updated as well.

Reply:
Done. Good catch, thanks!